packages/engine/tests/units/data/json/mod.rs

```diff
@@ -9,5 +9,5 @@ mod py {
     use crate::run_test;
 
     run_test!(access, Python);
-    run_test!(immutable, Python);
+    run_test!(immutable, Python, #[ignore]); // TODO
```

Comment:
Could you elaborate on the TODO (like in other tests)?

Reply:
```suggestion
    // TODO - test disabled until we make it so that the data actually is immutable
    run_test!(immutable, Python, #[ignore]);
```